tox.ini

```diff
@@ -6,6 +6,11 @@ envlist =
     mixtests
     evmtests
 
+[testenv]
+passenv =
+    GITHUB_TOKEN
```

Comment:
It's an issue with Travis, the Solidity Github API doesn't allow them to call in and so `py-solc-x` can't check for available versions.  You (probably) don't need to set it because your IP isn't banned.